test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Enums;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Enums
+{
+    [TestClass]
+    public class OpenApiSecuritySchemeTypeTests
+    {
+        [DataTestMethod]
+        [DataRow("None")]
+        [DataRow("Basic")]
+        [DataRow("Bearer")]
+        public void Given_Enum_Should_Have_Member(string memberName)
+        {
+            var members = typeof(OpenApiSecuritySchemeType).GetMembers().Select(p => p.Name);
+
+            members.Should().Contain(memberName);
+        }
+
+        [DataTestMethod]
+        //[DataRow("None", "")]
```

Comment:
Also we need another test code to test this enum value. Could you add another test method to ensure it?

Reply:
I added a new test method for None:
```
public void Given_Enum_Should_Have_No_Decorator(string memberName, string displayName)
```